Classes/Mapper/AssetModelMetaDataMapper.php

```diff
@@ -83,7 +103,7 @@ public function mapMetaData(Asset $asset, MetaDataCollection $metaDataCollection
         $contextVariables = array_merge($this->defaultContextVariables, $metaDataCollection->toArray());
 
         if (isset($this->metaDataMappingConfiguration['title'])) {
-            $asset->setTitle((string)EelUtility::evaluateEelExpression($this->metaDataMappingConfiguration['title'], $this->eelEvaluator, $contextVariables));
+            $asset->setTitle(substr((string)EelUtility::evaluateEelExpression($this->metaDataMappingConfiguration['title'], $this->eelEvaluator, $contextVariables), 0, 255));
```

Comment:
But it shouldn't be adjustable since the `title` cannot be longer than 255 chars (see [TYPO3\Media\Domain\Model\Asset](https://github.com/neos/media/blob/master/Classes/TYPO3/Media/Domain/Model/Asset.php#L81)).